charts/pega/templates/_pega-k8s-ingress.tpl

```diff
@@ -13,7 +13,11 @@ metadata:
 {{- end }}
 {{- else }}
     # Ingress class used is 'traefik'
-    kubernetes.io/ingress.class: traefik
+    kubernetes.io/ingress.class: {{ include "ingressClass" . }}
```

Comment:
I recommend creating AKS specific ingress tpl instead of conditioning in k8s-ingress tpl as we have provider specific ingress templates for GKE & EKS. To re-use the code blocks, consider generating below piece of code from helpers tpl. This seems to be common across all providers
```
 rules:
  # The calls will be redirected from {{ .node.domain }} to below mentioned backend serviceName and servicePort.
  # To access the below service, along with {{ .node.domain }}, http/https port also has to be provided in the URL.
  - host: {{ .node.service.domain }}
    http:
      paths:
      - backend:
          serviceName: {{ .name }}
          servicePort: {{ .node.service.port }}
```

Reply:
There are significant changes for other ingress definitions, but, for aks, this seems to be the only change.  Hence, we went with using the same file.  Better yet, I will create a new file and move it since it seems to be the common approach.